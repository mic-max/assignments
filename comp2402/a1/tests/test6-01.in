a2
a2
b2
b2
c3
c3
c3
d4
d4
d4
d4
e5
e5
e5
e5
e5