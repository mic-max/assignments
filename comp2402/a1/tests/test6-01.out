e5
d4
c3
a2
b2
